models/migrations/v78.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/go-xorm/xorm"
+)
+
+func renameRepoIsBareToIsEmpty(x *xorm.Engine) error {
+	type Repository struct {
+		ID      int64 `xorm:"pk autoincr"`
+		IsBare  bool
+		IsEmpty bool `xorm:"INDEX"`
+	}
+
+	if err := x.Sync(new(Repository)); err != nil {
+		return fmt.Errorf("sync: %v", err)
+	}
+
+	sess := x.NewSession()
+	defer sess.Close()
+	if err := sess.Begin(); err != nil {
+		return err
+	}
+
+	_, err := sess.Query("UPDATE `repository` SET `is_empty` = `is_bare`")
```

Comment:
Just putting a note here in case we have problems in the future, I've tested this statement sucessfuly on SQLite, and MySQL, we probably won't run into problems with this raw SQL on Postgres, and hopefully we won't run into it on MSSQL (can't test that DB right now sadly).

If errors do arise from this statement, then we'll have to select rows in batches and loop while doing updates on each using XORM `Update` function.